packages/js/product-editor/src/components/prepublish-panel/schedule-section/schedule-section.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import { PanelBody } from '@wordpress/components';
+import { useEntityProp } from '@wordpress/core-data';
+import {
+	DateSettings,
+	dateI18n,
+	getDate,
+	__experimentalGetSettings as getSettings,
+} from '@wordpress/date';
+import { createElement } from '@wordpress/element';
+import { __, _x, isRTL, sprintf } from '@wordpress/i18n';
+import {
+	// @ts-expect-error no exported member
+	__experimentalPublishDateTimePicker as PublishDateTimePicker,
+} from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { ScheduleSectionProps } from './types';
+import {
+	getSiteSettingsTimezoneAbbreviation,
+	isSameDay,
+	isSiteSettingsTime12HourFormatted,
+	isSiteSettingsTimezoneSameAsDateTimezone,
+} from '../../../utils';
+
+export function getFormattedDateTime( value: string ) {
```

Comment:
Out of curiosity, is there a specific reason for not creating a `getFormattedDateTime` utility under `utils/date`?

Reply:
Well, this is very specific to the schedule section. That is the only reason.